INSTALL.md

```diff
@@ -26,9 +26,11 @@ NEW or CHANGED MINIMUM dependencies since the last major release are **bold**.
  * **OpenColorIO >= 2.2** (tested through 2.3 and main)
  * libjpeg >= 8 (tested through jpeg9e), or **libjpeg-turbo >= 2.1** (tested
    through 3.0)
- * **[fmtlib](https://github.com/fmtlib/fmt) >= 7.0** (tested through 10.2).
+ * **[fmtlib](https://github.com/fmtlib/fmt) >= 7.0 < 11**
```

Comment:
You've changed the meaning of this line. "Tested through 10.2" is not just a claim about what versions we think we support, it's making a more exact statement about what the highest version is that is tested by our CI.

Reply:
Yes, you are right. I found that v11 was incompatible, so tried to make it more restrictive, but by doing so, information is lost.

Reply:
Oh no, I had not realized that version 11 had been released!  It's probably not really incompatible, this rejection is only because we say
```
checked_find_package (fmt REQUIRED
                      VERSION_MIN 7.0
                      VERSION_MAX 10.99
                      BUILD_LOCAL missing
                     )
```

I don't suppose you're interested in trying out doing this fix yourself?

- [ ] Edit the lines above in externalpackages.cmake to raise VERSION_MAX.
- [ ] Edit the CI "linux-latest-releases" test to specify the latest release.
- [ ] Push to your branch, triggering CI to verify that this all works.
- [ ] (Hopefully unnecessary) Fix anything that's broken!
- [ ] Submit PR.

Reply:
Wait, there is something that will need fixing, and it's not at all obvious. I'll take care of this one.